lib/java/com/google/android/material/slider/Slider.java

```diff
@@ -572,6 +583,15 @@ public void setOnChangeListener(@Nullable OnChangeListener listener) {
     this.listener = listener;
   }
 
+  /**
+   * Registers a callback to be invoked when the slider touch event is being started or stopped
+   *
+   * @param listener The callback to run when the slider starts or stops being touched
+   */
+  public void setOnSliderTouchListener(@Nullable OnSliderTouchListener listener) {
```

Comment:
Could we keep a list with add and remove methods instead? I'm planning to make that same change to the `OnChangeListener`.

Reply:
Yes, I can do this :)

Reply:
Done. I've also done it for OnChangeListener since it's similar